UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs

```diff
@@ -52,19 +52,23 @@ public async Task<string> CreateExchangeSet(SalesCatalogueServiceResponseQueueMe
                 });
                 await Task.WhenAll(tasks);
             }
+            else
```

Comment:
This else block is not required.

Reply:
As per you comments I have removed else block which is not required.